### PR TITLE
Introduce new DPTs for metering

### PR DESCRIPTION
**Pull request by holgerfriedrich**
Add dpt ids for 13.1200, 13.1201, 14.1200, 14.1201.
Change unit of dpt 12.1200 from litre to l.

Signed-off-by: Holger Friedrich <mail@holger-friedrich.de>

### CODE REVIEW

File: src/tuwien/auto/calimero/dptxlator/DPTXlator4ByteFloat.java [hunk 1]
```diff
@@ -610,6 +610,20 @@ public class DPTXlator4ByteFloat extends DPTXlator
 	/** DPT ID 14.080, Apparent Power; values from <b>-3.40282347e+38f</b> to <b>3.40282347e+38f</b> VA. */
 	public static final DPT DPT_APPARENT_POWER = new DPT("14.080", "Apparent Power", "-3.40282347e+38f",
 			"3.40282347e+38f", "VA");
+	// metering
+
+	/**
+	 * DPT ID 14.1200, volume flux [m³/h] for meters; values from <b>-3.40282347e+38f</b> to
+	 * <b>3.40282347e+38f</b> liter.
```
Comment (bmalinowsky) [inline src/tuwien/auto/calimero/dptxlator/DPTXlator4ByteFloat.java:617]: Unit is not liter here.

File: src/tuwien/auto/calimero/dptxlator/DPTXlator4ByteFloat.java [hunk 2]
```diff
@@ -610,6 +610,20 @@ public class DPTXlator4ByteFloat extends DPTXlator
...
+	 */
+	public static final DPT DPT_VOLUME_FLUX_METER = new DPT("14.1200", "volume flux [m³/h]", "-3.40282347e+38f", "3.40282347e+38f", "m³/h");
+
+	/**
+	 * DPT ID 14.1201, volume flux [l/s] for meters; values from <b>--3.40282347e+38f</b> to
```
Comment (bmalinowsky) [inline src/tuwien/auto/calimero/dptxlator/DPTXlator4ByteFloat.java:622]: There are 2 minus signs

File: src/tuwien/auto/calimero/dptxlator/DPTXlator4ByteFloat.java [hunk 3]
```diff
@@ -610,6 +610,20 @@ public class DPTXlator4ByteFloat extends DPTXlator
...
+	 * <b>3.40282347e+38f</b> m³.
```
Comment (bmalinowsky) [inline src/tuwien/auto/calimero/dptxlator/DPTXlator4ByteFloat.java:623]: Unit is not cubic meters.

File: src/tuwien/auto/calimero/dptxlator/DPTXlator4ByteSigned.java [hunk 4]
```diff
@@ -120,6 +120,20 @@ public class DPTXlator4ByteSigned extends DPTXlator
 	 */
 	public static final DPT DPT_DELTA_TIME = new DPT("13.100", "time lag", "-2147483648", "2147483647", "s");
 
+	// metering
+
+	/**
+	 * DPT ID 13.1200, volume liquid [l] for water/heat meter total consumption; values from <b>-2147483648</b> to
+	 * <b>2147483647</b> liter.
+	 */
+	public static final DPT DPT_DELTA_VOLUME_LIQUID_LITRE = new DPT("13.1200", "volume liquid [l]", "-2147483648", "2147483647", "l");
```
Comment (bmalinowsky) [inline src/tuwien/auto/calimero/dptxlator/DPTXlator4ByteSigned.java:129]: LITRE -> LITER

Comment (holgerfriedrich) [reply src/tuwien/auto/calimero/dptxlator/DPTXlator4ByteSigned.java:129]: this is actually how KNX.org defined it.... https://www.knx.org/wAssets/docs/downloads/Certification/Interworking-Datapoint-types/03_07_02-Datapoint-Types-v02.02.01-AS.pdf
I can change if you want, for me it sounds unfamiliar as well....

File: src/tuwien/auto/calimero/dptxlator/DPTXlator4ByteFloat.java (hunk 3 above)
Comment (holgerfriedrich) [reply src/tuwien/auto/calimero/dptxlator/DPTXlator4ByteFloat.java:623]: will change

File: src/tuwien/auto/calimero/dptxlator/DPTXlator4ByteFloat.java (hunk 2 above)
Comment (holgerfriedrich) [reply src/tuwien/auto/calimero/dptxlator/DPTXlator4ByteFloat.java:622]: will change

File: src/tuwien/auto/calimero/dptxlator/DPTXlator4ByteSigned.java (hunk 4 above)
Comment (bmalinowsky) [reply src/tuwien/auto/calimero/dptxlator/DPTXlator4ByteSigned.java:129]: They just used BE, not AE (english).

Comment (holgerfriedrich) [reply src/tuwien/auto/calimero/dptxlator/DPTXlator4ByteSigned.java:129]: Shall I change it as suggested?

Comment (bmalinowsky) [reply src/tuwien/auto/calimero/dptxlator/DPTXlator4ByteSigned.java:129]: Yes, please (liter is used everywhere else in the source code)

Comment (holgerfriedrich) [reply src/tuwien/auto/calimero/dptxlator/DPTXlator4ByteSigned.java]: done

**Comment by bmalinowsky** [comment]
Thanks!

**Pull request merged**